include/rocksdb/write_batch.h

```diff
@@ -25,10 +25,13 @@
 #pragma once
 
 #include <stdint.h>
+
 #include <atomic>
+#include <functional>
```

Comment:
Is this needed if not using std::function? (Would std::function be better?)

Reply:
I can remove `#include<functional>`.